isis/src/base/apps/shadow/shadow.xml

```diff
@@ -20,7 +20,8 @@
 
     <strong>User-Requirements</strong><br/>
       The user must supply an elevation model (DEM) and either an observation time or a cube
-      with raw camera geometry (see 'spiceinit').<br/><br/>
+      with raw camera geometry (see 'spiceinit'). If providing an observation time, the SPICE kernels
+      associated with the image target must have been downloaded and be available on the user's machine.<br/><br/>

```

Comment:
Since you just saw the code, it would be nice to specify which kernels if it can be boiled down. I don't mean file names, more like file types, and might want to use $ISISDATA in the description